### ISSUE TITLE
Radioboxes are not keyboard-accessible

### ISSUE THREAD

**Issue by lgarron** [720kb/radiobox.css#3]
The UI element should be in the normal tab order, highlighted on focus, and triggered on enter.

**Comment by 45kb** [comment]
Hi, the demo page you mean?

**Comment by lgarron** [comment]
> Hi, the demo page you mean?

I guess. If the demo page doesn't handle it properly, I'm going to assume the library doesn't do it properly.

**Comment by 45kb** [comment]
Of course i'll check that asap, thanks!

**Comment by 45kb** [comment]
Should work now, tab until you focus the first radio then space to check it and left and right arrows to navigate all them.

Let know if still any trouble  👍

**Comment by lgarron** [comment]
Works pretty well, thanks!
